### ISSUE TITLE
Progress bar on web UI seems broken

### ISSUE THREAD

**Issue by TimJones** [DoubleUTH/DSP-Seed-Finder#2]
Hi, great project btw.

I was trying to use the Galaxy Finder for a new play through and noticed the progress bar doesn't seem to advance at all. Not a front-end developer though so not sure what the issue could be...

![image](https://github.com/DoubleUTH/DSP-Seed-Finder/assets/113170/02c87920-e76a-4844-a9b6-11a36a1de4e7)

**Comment by DoubleUTH** [comment]
TL;DR: The concurrency settings on your PC are set too high. Using the default settings should resolve the issue.

Concurrency refers to the number of processes that run simultaneously when searching for seeds. In your case, a concurrency value of 32 is likely higher than the maximum supported concurrency of your PC. Typically, this maximum value is equal to the number of CPUs (doubled for dual-core, quadrupled for quad-core, etc.). The problem you encountered is that one of the processes responsible for processing seed 10 became stuck due to high CPU usage. Therefore, the progress bar appears to be broken.

To address this issue, it is recommended to use the default concurrency settings, which are adjusted to match the maximum concurrency supported by your PC. Unless the default settings are not utilizing 100% of your CPU, increasing the concurrency would likely result in decreased performance.

**Comment by TimJones** [comment]
Ah OK. I set it to 32 as I have a SMT-capable CPU and usually can crank concurrency up, but I was surprised by one thread being CPU-bound would block the UI thread or any of the threads not sharing the same core. Setting it back to the default 16 did work though, so thanks for the info!
